### ISSUE TITLE
Encoding gets lost when copy/pasting a file

### ISSUE THREAD

**Issue by jrieken** [Microsoft/vscode#44505]
re #44292

* in window 1, have a file with `hellö world`
* save that file with 'windows 1256' encoding
* copy that file into window 2
* in window 2, reopen that file (from utf8) to windows 1256
* 🐛 the ö got lost

**Comment by jrieken** [comment]
Hm, maybe an issue with that encoding...

**Comment by bpasero** [comment]
Yes, this is because we do not remember the encoding for files when closing and reopening them.

**Comment by jrieken** [comment]
>  closing and reopening them

No, it's not that. When I do re-open with encoding it's garbage, likely because the character cannot be encoded

**Comment by bpasero** [comment]
@jrieken ah yeah, it seems what is needed is a way to find out if the characters in the document can be encoded or not with the selected encoding. Similar to this:

![image](https://user-images.githubusercontent.com/900690/36909026-39bd118a-1e3d-11e8-9c76-9c4a2c12bb0a.png)

**Comment by bpasero** [comment]
Depends on https://github.com/ashtuchkin/iconv-lite/issues/53

**Comment by worc4021** [comment]
Is there any news on this? Find and replace becomes a lot less useful when you have to manually check the files for e.g. the degree symbol ° afterwards to make sure you won't bomb any application that will read that file in and trip over some hieroglyph.
